### DISCUSSION TITLE
Combining Motion Matching and Orientation Prediction for Godot 4 VR devices

### DISCUSSION THREAD

**Discussion by GeorgeS2019** [GuilhermeGSousa/godot-motion-matching#30] [Ideas]
Combining Motion Matching and Orientation Prediction to Animate Avatars for Consumer-Grade VR Devices

https://github.com/UPC-ViRVIG/MMVR

@Malcolmnixon
@BastiaanOlij

![image](https://github.com/user-attachments/assets/9461747e-9a97-4188-a424-4767c3bfad16)

![image](https://github.com/user-attachments/assets/26608429-4ca8-4652-bb49-c98476d62a65)

![image](https://github.com/user-attachments/assets/b1a57ed0-728a-4bb4-bf2d-17ba9e16562c)

**Comment by BastiaanOlij** [comment]
Owh thats cool!

**Comment by GeorgeS2019** [comment]
@GuilhermeGSousa 

The real time motion capture using Mediapipe is almost there
https://github.com/srcnalt/rpm-godot-sdk/issues/1
